nemo/collections/cv/models/visual_rnnt_bpe_models.py

```diff
@@ -0,0 +1,321 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import os
+from typing import Dict, List, Optional, Union
+
+import torch
+from omegaconf import DictConfig, ListConfig, OmegaConf, open_dict
+from pytorch_lightning import Trainer
+
+from nemo.collections.asr.losses.rnnt import RNNTLoss
+from nemo.collections.asr.metrics.rnnt_wer_bpe import RNNTBPEWER, RNNTBPEDecoding, RNNTBPEDecodingConfig
+from nemo.collections.asr.parts.mixins import ASRBPEMixin
+from nemo.collections.cv.data import video_to_text_dataset
+from nemo.collections.cv.models.visual_rnnt_models import VisualEncDecRNNTModel
+from nemo.core.classes.common import PretrainedModelInfo
+from nemo.utils import logging, model_utils
+
+
+class VisualEncDecRNNTBPEModel(VisualEncDecRNNTModel, ASRBPEMixin):
+    """Base class for encoder decoder RNNT-based models with subword tokenization."""
+
+    @classmethod
+    def list_available_models(cls) -> List[PretrainedModelInfo]:
```

Comment:
Could you move this to the end of the class?

Reply:
done